cmd/podman/images/build.go

```diff
@@ -44,6 +44,7 @@ var (
 	// Command: podman _diff_ Object_ID
 	buildDescription = "Builds an OCI or Docker image using instructions from one or more Containerfiles and a specified build context directory."
 	buildCmd         = &cobra.Command{
+		Aliases:           []string{"buildx"},
```

Comment:
I think the alias should be `buildx build` (buildx has other subcommands) so when building an image with `docker`, one would call `docker buildx build ...`

Reply:
Yes it has to be `podman buildx build`, other buildx subcommands should raise an error, see https://docs.docker.com/buildx/working-with-buildx/.